.github/workflows/aws_tfhe_gpu_pcc.yml

```diff
@@ -0,0 +1,126 @@
+# Perfom tfhe-cuda-backend post-commit checks on an AWS instance
+name: TFHE Cuda Backend - Post-commit Checks
```

Comment:
is this split still meaningful ? I think the gpu pcc executes first now and is a dedicated job ?

Reply:
Yes it is regarding parallelization.